packages/server/modules/shared/helpers/envHelper.ts

```diff
@@ -374,7 +374,7 @@ export function isEmailEnabled() {
 }
 
 export function postgresMaxConnections() {
-  return getIntFromEnv('POSTGRES_MAX_CONNECTIONS_SERVER', '4')
+  return getIntFromEnv('POSTGRES_MAX_CONNECTIONS_SERVER', '8')
```

Comment:
i was running into connection errors just because of a few assignToWorkspace() calls wrapped in a Promise.all. 4 is a ridiculously small number, and 8 is also, but it does resolve the issue I had